<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskswindow</class>
 <widget class="QDialog" name="taskswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задания</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: #2c3e50;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Выберите тему</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
       color: #000000;
       background-color: #ffffff; 
}
QComboBox QAbstractItemView {
       color: #000000;
       background-color: #ffffff;
       selection-background-color: #3498db;
       selection-color: white;
}</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: black;              
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #27ae60;
}
</string>
        </property>
        <property name="text">
         <string>Посмотреть задания</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>280</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #000000;background-color: #ffffff;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #3498db; 
    color: black;           
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #2980b9;
}</string>
        </property>
        <property name="text">
         <string>Предыдущее задание</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #3498db; 
    color: black;           
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #2980b9;
}</string>
        </property>
        <property name="text">
         <string>Следующее задание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: black;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: #2c3e50;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Ответ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #000000;background-color: #ffffff;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: black;              
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #27ae60;
}
</string>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QPushButton" name="pushButton_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #e74c3c; 
    color: black;               
    border-radius: 5px;      
}
QPushButton:hover {
    background-color: #c0392b; 
}</string>
     </property>
     <property name="text">
      <string>Главное меню</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
